additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py

```diff
@@ -598,7 +600,10 @@ async def inner_test_create_and_cancel_limit_orders(self, symbol=None, settlemen
         # )
         # # end debug tools
         open_orders = await self.get_open_orders(exchange_data)
-        cancelled_orders = await self.get_cancelled_orders(exchange_data)
+        try:
+            cancelled_orders = await self.get_cancelled_orders(exchange_data)
+        except trading_errors.NotSupported as err:
```

Comment:
This should not be necessary when setting `SUPPORT_FETCHING_CANCELLED_ORDERS=False` in exchange test.
It will also enable to run `test_get_cancelled_orders` for this exchange (instead of passing), the `test_get_cancelled_orders` will then make sure that `get_cancelled_orders` is really raising this error